cli/notifications_cli.rb

```diff
@@ -11,4 +11,9 @@ def import_message_batch(message_template_name, recipients_csv)
   rescue Exception => e
     puts e.message
   end
+
+  desc "send_test_message to_number message_body", "Sends a single SMS containing message_body to to_number. e.g. thor message:send_test_message +15555555555 'Hello, this is phone'"
```

Comment:
From checking this out and trying it, it seems that the invocation instructions here are no longer accurate (and I don't know how to invoke this now). Can you move general invocations instructions for commands in this file to a comment at the top of the file, or into the README?